tests/config-import.js

```diff
@@ -212,6 +212,22 @@ test('config import - presets', async (t) => {
   t.is(config.warnings.length, 0, `no warnings on the file`)
 })
 
+test('config import - translations', async (t) => {
+  const config = await readConfig(defaultConfigPath)
+  for (let { _, value: translationValue } of config.translations()) {
```

Comment:
Two small nitpicks: (1) we don't need `_` (2) we should use `const` instead of `let` because we aren't changing it.

```suggestion
  for (const { value: translationValue } of config.translations()) {
```